src/goose/console/views/redis/layout.clj

```diff
@@ -0,0 +1,26 @@
+(ns goose.console.views.redis.layout
+    (:require [hiccup.page :refer [include-css html5]]))
+
+(defn application [& components]
```

Comment:
This isn't redis specific, could move to a more neutral place.

Reply:
The layout consisted of `nav-bar` which varies with brokers.
But, I have separated the navbar from the application function and moved it to views folder (more neutral place).